### ISSUE TITLE
Training loss does not change

### ISSUE THREAD

**Issue by tessavdheiden** [agrimgupta92/sgan#2]
Hi Agrim,

The description of the code is super and working with the pretrained models as well!

However, I am trying to train it myself, but the generator's and discriminator's loss do not drop. Can you please let me know if the loss changed for your setting?

Best
Tessa

**Comment by agrimgupta92** [comment]
Try using print args scripts on one of the pre trained models and then use the same hyper parameter for your training. I would also suggest looking at weight initializations they helped with training.

**Comment by axem20** [comment]
Hi Tessa,

I'm having the same problem now. I tried using the same hyper parameters as for the zara1_12 model, however it didn't help. Do you remember how you fixed the issue?

Many thanks
